batch/batch/globals.py

```diff
@@ -16,6 +16,7 @@
 
 HTTP_CLIENT_MAX_SIZE = 8 * 1024 * 1024
 
-BATCH_FORMAT_VERSION = 2
-STATUS_FORMAT_VERSION = 2
-INSTANCE_VERSION = 3
+BATCH_FORMAT_VERSION = 3
+STATUS_FORMAT_VERSION = 3
+INSTANCE_VERSION = 4
```

Comment:
It's used to know what version of the instances we have in our current worker pool. We have to increment the version because they're sending a slightly different status to the driver.